### ISSUE TITLE
no information here when following help link

### ISSUE THREAD

**Issue by mfrasca** [maproulette/maproulette2#609]
https://maproulette.org/admin/project/4816/challenge/9054/edit
links to 
https://github.com/maproulette/maproulette2/wiki/Introducing-New-MapRoulette---Part-1.-Creating-and-Maintaining-Challenges#via-overpass
which says:
No information here

**Comment by mvexel** [comment]
Good point. This is a MapRoulette front end issue, so I'll move it to MR3
